security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -0,0 +1,48 @@
+:_content-type: ASSEMBLY
+[id="cert-manager-operator-release-notes"]
+= {cert-manager-operator} release notes
+include::_attributes/common-attributes.adoc[]
+:context: cert-manager-operator-release-notes
+
+toc::[]
+
+The {cert-manager-operator} is a cluster-wide service that provides application certificate lifecycle management.
+
+These release notes track the development of {cert-manager-operator}.
+
+:FeatureName: The {cert-manager-operator}
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+For more information, see xref:../../security/cert_manager_operator/index.adoc#cert-manager-operator-about[About the {cert-manager-operator}].
+
+[id="cert-manager-operator-release-notes-1.7.1"]
+== Release notes for {cert-manager-operator} 1.7.1 (Technology Preview)
+
+// TODO: Update if needed
+Issued: 2022-03-29
+
+The following advisory is available for the {cert-manager-operator} 1.7.1:
+
+// TODO: Replace link and name
+* link:https://access.redhat.com/errata/RHEA-TODO[RHEA-TODO - TODO]
```

Comment:
@bergerhoffer I got a '404' error when trying to access this link. I tried it twice, but check to make sure it resolves.

Reply:
@tmalove sorry! I should have mentioned that this link won't work yet - it'll get replaced with a real advisory once we know the link.